src/apps/loyalty/containers/thankyou/index.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react"
+
+import Button from "../../../../components/buttons/inverted"
+import Nav from "../../../../components/nav"
+import NavItem from "../../../../components/nav_item"
+import Text from "../../../../components/text"
+import TextLink from "../../../../components/text_link"
+import Title from "../../../../components/title"
+
+class Thankyou extends React.Component<any, any> {
```

Comment:
You are right, I didn't notice I could use a stateless component. I'll change it 👍